assets/img/favicon/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+ "name": "App",
+ "icons": [
+  {
+   "src": "\/android-icon-36x36.png",
```

Comment:
the `\/andorid....` looks weird. Should it be escaped like that?

Reply:
I've seen it both ways. But, these paths aren't accurate either. I'll need to edit this regardless.

Reply:
Pushed an update